test/multiverse/suites/grpc/Envfile

```diff
@@ -1,6 +1,9 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
-suite_condition('grpc gem requires CRuby 2.5+') do
+suite_condition('grpc gem requires CRuby (no other Rubies), at version 2.5+') do
```

Comment:
- gRPC `Envfile`: make it more clear that the CRuby requirement excludes other
  Rubies (JRuby, Truffle, mruby, etc.), regardless of their CRuby version
  compatibility (the `grpc` gem uses C extensions and does not offer .jar
  library equivalents)